src/main/java/edu/cse231/models/Order.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
+ */
+package edu.cse231.models;
+import java.util.*;
+/**
+ *
+ * @author Admin
+ */
+public class Order {
+     private String id;
+    private String customerName;
+    private Date date;
+    private List<OrderItem> items;
+    private Recipe recipe; // Assume Recipe class is defined
+    private Payment payment; // Assume Payment class is defined
+
+    public Order(String id, String customerName, Date date) {
+        this.id = id;
+        this.customerName = customerName;
+        this.date = date;
+        this.items = new ArrayList<>();
+    }
+
+    public String getId() {
+        return id;
+    }
+
+    public String getCustomerName() {
+        return customerName;
+    }
+
+    public Date getDate() {
```

Comment:
هنا بس ممكن نخلي ال date البيرجع يكون copy عشان انت بترجع reference 
```suggestion
    public Date getDate() {
      return new Date(date.getTime());
    }
```